operator/src/main/resources/scripts/livenessProbe.sh

```diff
@@ -35,9 +35,9 @@ if [ ! "${DYNAMIC_CONFIG_OVERRIDE:-notset}" = notset ]; then
       tgt_file=$tgt_dir/$tgt_file             # add back in tgt dir path
```

Comment:
Minor comment:

The 'copyIfChanged' looks like it already contains the diff/continue logic on lines 36 & the chmod logic on lines 39,40, and 41.  I think you can you go ahead and delete these lines.

On the other hand, copyIfChanged is a little talkative - it traces even when it decides to skip the copy.  This might be a little distracting since the liveness probe is frequently invoked.